src/Event/EventManager.php

```diff
@@ -485,6 +485,7 @@ public function __debugInfo(): array
         if ($this->_eventList) {
             $count = count($this->_eventList);
             for ($i = 0; $i < $count; $i++) {
+                /** @var \Cake\Event\EventInterface $event */
```

Comment:
Too bad phpstan can't infer the type in spite of https://github.com/cakephp/cakephp/blob/918ecc19ab4c3e2bac0fe45638173cb08d6677b5/src/Event/EventList.php#L25

Reply:
For some issues we could raise a ticket in phpstan repo, maybe they will fix up those things in future versions.
Then we can drop some annotations again.